packages/cli/src/index.ts

```diff
@@ -1,9 +1,24 @@
+import { existsSync } from "fs-extra";
 import "dotenv/config";
 import yargs from "yargs/yargs";
 import * as preview from "./commands/preview";
 import * as init from "./commands/init";
+import { looksLikeTypescriptProject, MAILING_CONFIG_FILE } from "./config";
 
+const fs = require("fs");
```

Comment:
switch to import syntax plz

Reply:
Weirdly this function behaves differently when imported (returns a stream instead of a string). I will muck with it a bit more to see what is the matter with it